sormas-backend/src/main/java/de/symeda/sormas/backend/common/AbstractDomainObject.java

```diff
@@ -181,6 +181,15 @@ public int hashCode() {
 		return getUuid().hashCode();
 	}
 
+	public String caption() {
```

Comment:
for me adding the @Transient was not enough, the server wouldn't start up by complaining that there is no such attribute

Reply:
it is used in some places when we build some string in the backend or to create ReferenceDtos
not sure how we should move it to DTOs, the backend has the api as dependency not the other way around
if @MateStrysewske agrees with you we can rework this

Reply:
Since the `caption()` method is in entities, it's part of the backend, so it can easily be moved to EJBs where `caption()` is currently invoked, and using existing helper methods there.

Reply:
Fine by me to put it into the EJBs